source/declarations.tex

```diff
@@ -2607,12 +2592,16 @@
 \begin{ncsimplebnf}
 \terminal{*} \opt{attribute-specifier-seq} \opt{cv-qualifier-seq} \terminal{D1}
 \end{ncsimplebnf}
-and the type of the identifier in the declaration
+and the type of the contained
+\grammarterm{declarator-id}
+in the declaration
 \tcode{T}
 \tcode{D1}
 is ``\placeholder{derived-declarator-type-list}
 \tcode{T}'',
```

Comment:
These, again, are two completely different `T`s. =(

Reply:
I think that there is no issue here - the point is to use the same decl-specifier-seq in invented declaration `T D1`.